pkg/utils/utils.go

```diff
@@ -921,6 +927,25 @@ func RunCommand(command string, args ...string) (string, error) {
 	return stdout.String(), err
 }
 
+func HashConfigMap(cm *corev1.ConfigMap) string {
+	var keys []string
+	for k := range cm.Data {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+
+	var concatenatedString string
```

Comment:
Using bytes.Buffer is more efficient than allocating a string in every iteration. It looks better as well. However, the performance gain should be negligible for the amount of data that this config map usually contains. So, ignore this comment if you don't want to change it.

Reply:
Pushed a change to optimize this even further.